nb-apis/nb-api/src/main/java/io/nosqlbench/nb/api/tagging/TagFilter.java

```diff
@@ -156,6 +156,9 @@ public TagFilter(String filterSpec) {
             for (String assignment : keyvalues) {
                 String[] keyvalue = assignment.split("[:=]", 2);
                 String key = keyvalue[0];
+                if (key.endsWith("s")) {
```

Comment:
omg, that's a clever idea 💡 !!

Reply:
hopefully we don't have any legitimate keys that ends with `s`!

Reply:
We don't - there seem to be three categories defined in `OpDef.java`
```
        tagsWithName.put("block",block.getName());
        tagsWithName.put("name",this.rawOpDef.getName());
        tagsWithName.put("op",this.rawOpDef.getName());
```